src/pipecat/services/gladia.py

```diff
@@ -37,17 +37,17 @@
 def language_to_gladia_language(language: Language) -> str | None:
     match language:
         case Language.BG:
-            return "bg"
+            return "bulgarian"
```

Comment:
Weird! For a websocket connection, they have very little documentation (that I can find). I found the language codes (ISO-639) for their REST endpoint, which listed the two letter codes. Maybe I'm just missing the docs. Thanks for catching this.